### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by JSYoo5B** [Hanjp-IM/hanjp-im#46]
Known problems:
1. Getting started is linking wrong url for cloning repository  
    It's linking https://github.com/ubuntu-kr/hanjp-im.git  
    (Seems this is legacy link for this project)
2. Need to describe **Build dependancy package**  
    (Also, It'd be better if you can describe how to install them by command line)

Recommendation
1. Move **About Hanjp project** and demo video at the top of README.md  
    As the README.md is the entry point of the projects, It'd be better to give information on its top
2. Describe currently supported platforms
    For end-user, supported platform may helpful for using Hanjp-IM
    Also it may invoke some minor & untracked platform users to contribute.
3. Separate **Getting started** into other document such as INSTALL.md  
    When the contents of README.md becomes longer, It'd be better to separate documents by its topic

Seems these issues are good first issue.

**Comment by JSYoo5B** [comment]
FYI, Build dependency packages in Ubuntu were following:
* libhangul-dev
* libibus-1.0-dev

**Comment by onting** [comment]
Good suggestions.
I reckon your suggestions are,
1. It is not a wrong link. The URL is renewing document link. Korean wiki page is outdated one. We are migrating all official documents to English. Because I believe that open source contribution without reading of English is very painful, paradoxically. I believe all of open source newcomers should be used to use of English. All official docs have to written in English. Additional supportive docs can be written in Korean for an ease of understanding and Korean community promotions.
2. That's right. The dependency must be noted on documents clearly. Good pointing out.
3. I think that is unnecessary. The quick starting guide should noted in most easily lookupable place.
Thanks.

**Comment by JSYoo5B** [comment]
Some issues are resolved by @onting in commit 6138d7807bae50c8c2f04d617effdf6f9f329e02

But still, I think there are more tasks to improve README.md

I'll yield these issues as **good first issue**

**Comment by onting** [comment]
Yup, improvement for README can be a good starting point, I also believe.
So, please feel free to talk with team.
